libraries/aws_vpc_endpoint.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsVpce < AwsResourceBase
+  name 'aws_vpc_endpoint'
+  desc 'Verifies settings for an AWS VPC Endpoint'
+
+  example "
+    describe aws_vpc_endpoint(vpc_endpoint_id: 'vpce-12345678123456789') do
+      it { should exist }
+    end
+  "
+
+  def initialize(opts = {})
+    opts = { vpc_endpoint_id: opts } if opts.is_a?(String)
+    super(opts)
+    validate_parameters(required: [:vpc_endpoint_id])
+
+    if opts[:vpc_endpoint_id] !~ /(^vpce\-[0-9a-f]{17}$)|(^vpce\-[0-9a-f]{8}$)/
+      raise ArgumentError, "#{@__resource_name__}: 'vpc_endpoint_id' must be in the format 'vpce-' followed by 8 hexadecimal characters."
+    end
+
+    @display_name = opts[:vpc_endpoint_id]
+    filter = { name: 'vpc-endpoint-id', values: [opts[:vpc_endpoint_id]] }
+    catch_aws_errors do
+      resp    = @aws.compute_client.describe_vpc_endpoints({ filters: [filter] })
```

Comment:
slight nitpick, but please remove the extra spacing.b

Reply:
✨ Done